roles/rhods_test_jupyterlab/tasks/main.yml

```diff
@@ -176,6 +176,43 @@
     oc delete ev -n {{ rhods_test_namespace }} --all > /dev/null
   failed_when: false
 
+- name: Retrieve the configmap containing the default notebook size
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+  command:
+    oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile > \
+      {{ artifact_extra_logs_dir }}/jh_global_profile.json
```

Comment:
This one is tricky, I took a while to understand it:
```
  command:
        oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile > \
          {{ artifact_extra_logs_dir }}/jh_global_profile.json
      
```
generates this command:
```
oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile > \{{ artifact_extra_logs_dir }}/jh_global_profile.json
```
which is wrong
while this one:
```
  command: |
        oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile > \
          {{ artifact_extra_logs_dir }}/jh_global_profile.json
```
generates this command:
```
        oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile > \
          {{ artifact_extra_logs_dir }}/jh_global_profile.json
```
which is correct

Basically, when `|` is used, the string is passed as is to `bash`,
when it's not used, it's `" ".join(lines)`

Reply:
Fixed, I think